docs/user-guide/api-mediation/api-gateway-configuration.md

```diff
@@ -212,8 +212,18 @@ Add the following properties to the file for the API Gateway:
     
 ## CORS handling
 
-By default, Cross-Origin Resource Sharing (CORS) is disabled in the API Gateway for the Gateway routes `api/v1/gateway/**`. To enable CORS at the service level, it is necessary to enable CORS in the Gateway. Use the following procedure to enable CORS.
-        
+You can enable Gateway to terminate CORS requests for itself and also for routed services. By default, Cross-Origin Resource Sharing (CORS) handling is disabled in the API Gateway for the Gateway routes `api/v1/gateway/**`. To enable CORS at the service level, it is necessary to enable CORS as stated here first. Individual services can control whether they want Gateway to handle CORS for them through the [Custom Metadata](../../extend/extend-apiml/custom-metadata.md) parameters.
```

Comment:
Can we state this as "it is necessary to enable CORS before ...". It is unclear what is meant be first. What needs to happen first?

Reply:
I've adjusted the text to be more clear. Please let me know your opinion. Thank you.